tensorflow_addons/optimizers/BUILD

```diff
@@ -156,3 +157,17 @@ py_test(
         ":optimizers",
     ],
 )
+
+
+
+py_test(
+    name = "discriminative_layer_training_test",
+    size = "small",
```

Comment:
Please make this `"medium"` so that the test has sufficient time

Reply:
will do thanks!